qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -7,6 +7,7 @@
  * Simple.
  * @version $Id$
  * @author John Smith (john@example.com)
+ * @checkstyle FinalLocalVariable (51 lines)
```

Comment:
@jrdalpra I'm the author of this file and my idea was to test all variable names here: final and non-final. As you see there are final variables there, they were added for a reason